lib/galaxy/tools/errors.py

```diff
@@ -133,5 +133,9 @@ def _send_report( self, user, email=None, message=None, **kwd ):
         else:
             to = to_address
         subject = "Galaxy tool error report from %s" % email
+        try:
+            subject = "%s (%s)" % ( subject, self.app.toolbox.get_tool( self.job.tool_id, self.job.tool_version ).old_id )
+        except:
```

Comment:
Can you make this at very least a ``except Exception:`` or even more preferably whatever specific exception results when an unknown tool id and version are found.

Bare ``except``s are an anti-pattern - https://docs.python.org/2/howto/doanddont.html#except - and we should eliminate them from Galaxy.